cypher/cypher-docs/src/docs/dev/ql/administration/security/admin-role-introduction.asciidoc

```diff
@@ -0,0 +1,116 @@
+The built-in role `admin` includes a number of privileges allowing users granted this role the ability to perform administrative tasks.
+These include the ability to perform the following classes of tasks:
+
+* Manage <<administration-security-users-and-roles, users and roles>>
+* Manage <<administration-databases, multiple databases>>
+* Manage <<administration-indexes, indexes>> and <<administration-constraints, constraints>>
+* Manage <<administration-security-administration-database-privileges, database security>>
+* Manage <<administration-security-administration-dbms-privileges, DBMS security>>
+
+These rights are conferred using privileges that can be  managed using `GRANT`, `DENY` and `REVOKE` commands, with the exception of the _DBMS Security_ privileges which are only available within the built-in `admin` role.
+
+As described in the section on <<administration-security-subgraph, sub-graph security>>, the `GRANT` command allows an administrator to grant a privilege to a role in order to access an entity.
+The `DENY` command allows an administrator to deny a privilege to a role in order to prevent access to an entity.
+The `REVOKE` command allows an administrator to remove a previously granted or denied privilege.
+The syntax is:
+
+.Privilege command syntax
+[options="header", width="100%", cols="3a,2"]
+|===
+| Command | Description
+
+| [source, cypher]
+GRANT database-privilege ON DATABASE dbname TO role
+| Grant a privilege to a role
+
+| [source, cypher]
+DENY database-privilege ON DATABASE dbname TO role
+| Deny a privilege to a role
+
+| [source, cypher]
+REVOKE GRANT database-privilege ON DATABASE dbname FROM role
+| Revoke a granted privilege from a role
+
+| [source, cypher]
+REVOKE DENY database-privilege ON DATABASE dbname FROM role
+| Revoke a denied privilege from a role
+
+| [source, cypher]
+REVOKE database-privilege ON DATABASE dbname FROM role
+| Revoke a granted or denied privilege from a role
+|===
+
+[NOTE]
+These commands are very similar to the <<administration-security-subgraph-introduction, graph-privileges>> except for the use of the term `DATABASE`, no `entity` and and the set of available _database-privileges_ differs from the _graph-privileges_.
+
+Where the components are:
+
+* _database-privilege_
+** `ACCESS`
++
+allows access for a specific database/graph
++
+** `START`
++
+allows the specified database to be started
++
+** `STOP`
++
+allows the specified database to be stopped
++
+** `CREATE INDEX`
++
+allows indexes to be created on the specified database.
++
+** `DROP INDEX`
++
+allows indexes to be deleted on the specified database.
++
+** `INDEX MANAGEMENT`
```

Comment:
The `INDEX MANAGEMENT`, `CONSTRAINT MANAGEMENT` and `NAME MANAGEMENT` has `MANAGEMENT` as optional